packages/backend-modules/call-to-actions/graphql/resolvers/CallToActionPayload.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  __resolveType: (cta, args, context) => {
+    if (cta.customComponent) {
+      return 'CallToActionComponentPayload'
+    }
+
+    return 'CallToActionBasicPayload'
+  },
+}
```

Comment:
Handler receives `payload` attribute only, not entire call-to-action row.

```suggestion
module.exports = {
  __resolveType: (payload, args, context) => {
    if (payload.customComponent) {
      return 'CallToActionComponentPayload'
    }

    return 'CallToActionBasicPayload'
  },
}
```